### DISCUSSION TITLE
How to sync savegames between PC and Android 12?

### DISCUSSION THREAD

**Discussion by EXCIDIVM** [diasurgical/DevilutionX#5331] [Q&A] [answered]
Hello. Anyone has a working setup to automatically synchronize their savegames between PC and Android?

I used to use Syncthing for this, but starting with Android 11, you can no longer access the/android/data folder where all the data including the savegames are located.
Is there a way to change the savegame location on Android so Syncthing can access it?

**Comment by AJenbo** [comment, marked as answer]
Starting with Android 11, it is no longer possible for the app to access data outside the android/data folder.

We do support cloud backup, so the save games should be available in a hidden folder in your Google Drive. I don't know if it can be easily accessed outside of Android.

**Comment by EXCIDIVM** [comment]
I haven't seen any cloud backup functionality. Would it be in the PC or in the Android version and how would I activate it?

**Reply by AJenbo** [reply]
It's part of the Android OS. DevilutionX simply tells Android what files should be backed up.

https://one.google.com/

**Reply by EXCIDIVM** [reply]
Ah that makes sense. I am on a degoogled device, but good for others!
